api/src/play_asset_pack_manager.gd

```diff
@@ -22,6 +22,7 @@
 # Functions and signals provided by the Android plugin will use camelCasing as 
 # the naming convention, since the plugin is written in Java.
 # ##############################################################################
+# warnings-disable
```

Comment:
Ideally this should be ```# warning-ignore:return_value_discarded, unused_signal```. However this syntax is not supported, so we have to disable all warnings.